### PR TITLE
Changes in input and output of nn

### PR DESCRIPTION
**Pull request by Postman47**
Hey, here are some changes for review:

-Added removal of incomplete days in order to achieve standardized network input
-Changed normalization so now it normalizes to global min and max
-Changed input of network so now it takes data from 3 full days
-Changed output of network so now it calculates avg temperature of the target day and if at any given point during the day wind speed exceeded 6 m/s
-Added custom MSE loss function so it accepts temperature +- 2 degrees
-Changed training loop, now it separates output for regression and classification, calculates losses separately and combines them
-Added accuracy calculation

### CODE REVIEW

File: nn.py
Comment (mwlodarzc) [inline nn.py:1]: This could be done with Torch.Tensor([...]).chunk(2, dim=-1), just for you to know.

**Review by mwlodarzc — APPROVED** [review]
Looks good!
Thanks for requesting these changes, all look solid!
